process/outportBlockConverter.go

```diff
@@ -152,6 +497,237 @@ func (o *outportBlockConverter) HandleShardOutportBlock(outportBlock *outport.Ou
 	return shardOutportBlock, nil
 }
 
+func (o *outportBlockConverter) copyTransactions(sourceTxs map[string]*outport.TxInfo, transactionPool *hyperOutportBlocks.TransactionPoolV2) error {
+	for txHash, txInfo := range sourceTxs {
+		destTxInfo := &hyperOutportBlocks.TxInfoV2{}
+
+		// TxInfo - Transaction
+		if txInfo.Transaction != nil {
```

Comment:
revert if condition to avoid indentation? if transaction == nil --> continue

Reply:
good idea.